templates/style/rustdoc.scss

```diff
@@ -50,8 +50,8 @@ div.rustdoc {
             &.mobile {
                 top: $top-navbar-height;
 
-                .sidebar-elems .show-it {
-                    top: 77px;
+                .sidebar-elems.show-it {
+                    top: 45px + $top-navbar-height;
```

Comment:
Where does 45px come from?

Reply:
That's the height of the rustdocs sidebar nav.